include/misc/dlist.h

```diff
@@ -191,7 +191,7 @@ static inline void sys_dlist_init(sys_dlist_t *list)
 	list->tail = (sys_dnode_t *)list;
 }
 
-#define SYS_DLIST_STATIC_INIT(ptr_to_list) {{(ptr_to_list)}, {(ptr_to_list)}}
+#define SYS_DLIST_STATIC_INIT(ptr_to_list) {{(ptr_to_list)}, {(ptr_to_list)} }
```

Comment:
should we also add a space in the beginning of the array? (so it looks "symmetric")

Reply:
ok

Reply:
...Without triggering yet another checkpatch warning, of course.